core/services/vrf/v2/integration_v2_test.go

```diff
@@ -321,7 +324,7 @@ func newVRFCoordinatorV2Universe(t *testing.T, key ethkey.KeyV2, numConsumers in
 		uint16(1),                              // minRequestConfirmations
 		uint32(2.5e6),                          // gas limit
 		uint32(60*60*24),                       // stalenessSeconds
-		uint32(vrf.GasAfterPaymentCalculation), // gasAfterPaymentCalculation
+		uint32(v22.GasAfterPaymentCalculation), // gasAfterPaymentCalculation
```

Comment:
`GasAfterPaymentCalculation` should probably be a function and no longer a constant.